lib/cylc/cfgspec/suite.py

```diff
@@ -315,6 +318,16 @@ def _coerce_final_cycletime( value, keys, args ):
                 'execution timeout'           : vdr( vtype='interval_minutes'),
                 'reset timer'                 : vdr( vtype='boolean', default=False ),
                 },
+            'events' : {
+                'handler'                     : vdr( vtype='string_list', default=[] ),
+                'handler events'              : vdr( vtype='string_list', default=[] ),
+                'handler delays'              : vdr( vtype='interval_minutes_list', default=[] ),
```

Comment:
This was changed in one of my latest changeset. I did have `retry delays` originally. However, it is recognised that it may sometimes be useful to run the event handler with an initial delay, so this is now specifying when to run the handler the 1st time as well as for retries. (The default is effectively `PT0S`.)